subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1011,18 +1011,18 @@ static void ticker_cb(u32_t ticks_at_expire, u32_t remainder, u16_t lazy,
 	if (!lll->is_hdcd)
 #endif /* CONFIG_BT_PERIPHERAL */
 	{
-		u8_t random_delay;
+		u16_t random_delay;
 		u32_t ret;
 
 		ull_entropy_get(sizeof(random_delay), &random_delay);
-		random_delay %= 10;
-		random_delay += 1U;
+		random_delay %= HAL_TICKER_US_TO_TICKS(10000);
```

Comment:
I don't think u16_t will work for sub-microsecond tick configurations. I suggest changing the type to u32_t, in which case it will work for us.